### ISSUE TITLE
SlevomatCodingStandard.Namespaces.UnusedUses false positive for grouped use declarations

### ISSUE THREAD

**Issue by InvisibleSmiley** [slevomat/coding-standard#1311]
PHP allows grouped use declarations:
https://www.php.net/manual/en/language.namespaces.importing.php#language.namespaces.importing.group

The UnusedUses sniff does not support/handle these cases correctly which results in false positives (and removals when using phpcbf).

Please note that grouped NS definitions may span multiple lines, especially the comma-separated parts.

Example:
```
<?php

declare(strict_types=1);

namespace foo\bar\baz;

class X
{
}

namespace test;

use foo\bar\{baz};

class Y extends X
{
}
```

**Comment by kukulich** [comment]
Yes, we don't support grouped declarations. We think it's wrong to use them.

**Comment by InvisibleSmiley** [comment]
Do you have a (non-autofixing) sniff then that just forbids them?

**Comment by kukulich** [comment]
There's `DisallowGroupUseSniff` but without autofixing.

**Comment by InvisibleSmiley** [comment]
OK got it. Unfortunately that doesn't prevent UnusedUsesSniff from autofixing the code to become invalid.
